modular_citadel/code/game/objects/items/melee/eutactic_blades.dm

```diff
@@ -117,6 +117,15 @@
 	desc = "The Non-Eutactic Blade utilizes a hardlight blade that is dynamically 'forged' on demand to create a deadly sharp edge that is unbreakable. This one seems to have a damaged handle and misaligned components, causing the blade to be unstable at best"
 	force_on = 15 //As strong a survival knife/bone dagger
 
+/obj/item/melee/transforming/energy/sword/cx/chaplain
+	name = "divine lightblade"
+	force_on = 20		//haha i'll regret this
+	block_chance = 50
```

Comment:
This is flat out better then claymore and other weapons

Reply:
Normal null rod is 18
And the highest block you get is 30 with the claymore

Reply:
think this'll be fine.